Makefile

```diff
@@ -46,6 +46,9 @@ remove-tmp:
 
 clean: destroy-box remove-tmp
 
+proxy:
+	consul intention create -token=master postgres-local postgres
```

Comment:
Keep in mind that this command is not idempotent. Could you test it? (Run 2-3 times)